currency_rate_update/views/res_config_settings.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+
+<record id="view_res_config_settings" model="ir.ui.view">
+    <field name="name">currency_rate_update.res.config.settings.form</field>
+    <field name="model">res.config.settings</field>
+    <field name="inherit_id" ref="account.res_config_settings_view_form"/>
+    <field name="arch" type="xml">
+        <field name="group_multi_currency" position="after">
```

Comment:
It would be better like this: 
`
        <field name="module_currency_rate_live" position="replace">
                <field name="auto_currency_up" class="oe_inline"/>
        </field>
`
(assuming nobody intends to install enterprise module if they have this module)

Reply:
Better to put it as invisible, because other views that use that field as reference can fail.